### PR TITLE
schedule updates: titles/topics and leads

### PR DESCRIPTION
**Pull request by JessicaS11**
Re-adds missing tutorial leads (now that they don't have to be in the teams yaml); removes Doing Good Science session; clarifies data integration tutorial titles

### CODE REVIEW

File: book/schedule.yaml [hunk 1]
```diff
@@ -68,7 +68,7 @@ tabs:
         title: ICESat-2 Data Access
         description:
         leads:
-          - Jane Koh
+          - Mikala Bieg
```
Comment (scottyhq) [inline book/schedule.yaml:71]:
```suggestion
          - Mikala Beig
```

File: book/schedule.yaml [hunk 2]
```diff
@@ -94,10 +94,10 @@ tabs:
         title: Geospatial Transformations
         description: Getting oriented and goals
         leads:
-          - Jane Koh
+          - Tyler Sutterley
       - time: 9:30 - 10:30
         location: Zoom
-        title: Data Integration with other products (Part I of II)
+        title: Integration of ICESat-2 with other data - Tools and techniques for working with other data
```
Comment (scottyhq) [inline book/schedule.yaml:100]:
```suggestion
        title: Data Integration - Tools and techniques
```

Comment (JessicaS11) [reply book/schedule.yaml:100]: @tsnow03 How does this change sound to you?

**Comment by aaarendt** [comment]
@JessicaS11 it seems after the meeting we're all in agreement on the schedule. Should we merge it now?

**Comment by JessicaS11** [comment]
@aaarendt Sure - I'd been giving @tsnow03 time to comment on the suggested title change. Once we decide on that it should be good to go.

File: book/schedule.yaml (hunk 2 above)
Comment (tsnow03) [reply book/schedule.yaml:107]: I think the part 1 title looks great. Maybe part 2 can be Data Integration - Advanced analyses. I'm not clear on how to change it myself.

Comment (JessicaS11) [reply book/schedule.yaml]: Thanks @tsnow03!

**Review by aaarendt — APPROVED** [review]

**Pull request merged**